no need to use setuptools_cython anymore

pypeng opened the pull request:
```
cython is working fine
```

pypeng commented:
need this: https://github.com/douban/greenify/pull/14

windreamer commented:
@youngsofun pls review

mckelvin commented:
It works fine on latest version of `setuptools`. So what's the minimum required version of `setuptools`? since old versions may break. And it would be helpful to warning something if `setuptools.__version__` is not expected.

windreamer commented:
@pypeng we should ensure `setuptools >= 0.18.0`, I think.  @mckelvin

youngsofun commented:
@pypeng  what's your opinion  about setuptools version?

pypeng commented:
let me find the minimum version for setuptools.

windreamer commented:
@pypeng 
cf: http://setuptools.readthedocs.io/en/latest/history.html#id174

windreamer commented on setup.py:
```diff
@@ -71,7 +64,7 @@ def run_tests(self):
     ],
     # Support for the basestring type is new in Cython 0.20.
-    setup_requires=["setuptools_cython", "Cython >= 0.20"],
+    install_requires=["Cython >= 0.20", "setuptools >= 18.0"],
```
seems this does not work for setuptools. maybe we should add an assert to check setuptools version?

windreamer replied:
OK, `setup_requires` seems working, `install_requires` doesn't ...

pypeng commented:
yes, setuptools==18.0 is the minimum required version. i've updated setup.py.

I also changed setup_requires to install_requires, which is required by pip: https://pip.pypa.io/en/stable/user_guide/?highlight=install_requires

windreamer commented:
@pypeng I think just a note in `README` is enough... all these hacks looks weird and confuse.

windreamer commented:
LGTM
